core/arch/arm/plat-d06/platform_config.h

```diff
@@ -12,4 +12,9 @@
 /* Make stacks aligned to data cache line length */
 #define STACK_ALIGNMENT		64
 
+/* UART */
+#define UART_BASE		0x2f8
```

Comment:
Where is this used?

Reply:
I' ve checked. UART_BASE is used in `lpc_uart.c`  `lpc_byte_read(paddr_t addr, uint8_t *data)` and `static void lpc_byte_write(paddr_t addr, uint8_t data)`

but you show me a good point, I will check all `macro definition` which maybe unused.